test/Microsoft.Azure.WebJobs.Host.UnitTests/Common/HostCallTestsWithRouteData.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Host.UnitTests
+{
+    // Unit test for exercising Host.Call passing route data. 
```

Comment:
nit: I'd also replace all this "route data" terminology and file name with "binding data". There are no routes here